internal/oci/runtime_vm.go

```diff
@@ -337,13 +337,22 @@ func (r *runtimeVM) execContainerCommon(c *Container, cmd []string, timeout int6
 	}
 	defer execIO.Close()
 
+	// chan to notify that can call runtime's CloseIO API
+	closeIOChan := make(chan bool)
```

Comment:
@liubin, please. bear with me here as it may be a silly question. :-)

We're creating the channel here in order to notify who? Would containerd-shim-kata-v2 be "listening for it" and when this channel is closed, then it'd trigger the closeIO?

Reply:
Hi @fidencio , containerd-shim-kata-v2 don't care and can't care about it. The chan will closed after `exec` and `start` RPCs called, and the chan is not used to notify anyone to do something, it only let the code in other goroutine to continue to run, before the chan is closed( == `exec` not `start`ed), the call to underly `CloseIO` is not permitted.